### PR TITLE
Add git-crypt package, version 0.6.0

### PR DESCRIPTION
**Pull request by Krakonos**
A git plugin for transparent file encryption; works well in msys environment.

### CODE REVIEW

File: git-crypt/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,20 @@
+pkgname="git-crypt"
```
Comment (Alexpux) [inline git-crypt/PKGBUILD]: Add header

Comment (Krakonos) [reply git-crypt/PKGBUILD]: Amended. I hope the header is just the note about maintainer, which I put myself in (I'm willing to take care of the package in the future).

**Comment by Krakonos** [comment]
Bump. I have updated the patch quite some time ago. Could you take another look?

Thanks,
Ladislav

**Comment by Alexpux** [comment]
@Krakonos I dont see where you specify prefix for installing?

**Comment by Krakonos** [comment]
@Alexpux You are right. 

I have amended the commit to set correct prefix. (Sorry it took me so long, I missed your comment).

**Pull request merged**